src/Processor.php

```diff
@@ -164,7 +181,7 @@ public function filterMultiArray(array $data)
             if (is_array($value)) {
                 $filteredSubArray = $this->filterMultiArray($value);
                 $filteredArray = array_merge($filteredArray, $filteredSubArray);
-            } elseif (str_contains($value, self::VAULT_KEYWORD)) {
+            } elseif (substr($value, 0, 5) === self::VAULT_KEYWORD) {
```

Comment:
You assume that the value is 5 characters. What if the self::VAULT_KEYWORD is pariser?
either change to 
`substr($value, 0, strlen(self::VAULT_KEYWORD)` or use `strpos($value, self::VAULT_KEWORD) ==0`